hapi-utils/src/test/java/com/hedera/services/throttles/DeterministicThrottleTest.java

```diff
@@ -302,8 +302,9 @@ void equalsWorks() {
 		final var throttle1 = DeterministicThrottle.withTpsAndBurstPeriodNamed(tps, 2, name);
 		final var bucketThrottle = BucketThrottle.withTps(tps);
 
-		assertTrue(throttle.equals(throttle1));
-		assertFalse(throttle.equals(null));
-		assertFalse(throttle.equals(bucketThrottle));
+		assertEquals(throttle, throttle1);
+		final var equalsForcedCallResult = throttle.equals(null);
+		assertFalse(equalsForcedCallResult);
```

Comment:
`assertNotNull(throttle);` would be easier to understand

Reply:
@anighanta, but they don't assert/run the same code. Please read my variable name!